src/server/app.ts

```diff
@@ -76,7 +76,7 @@ export default function createApp(serverSettings: ServerSettings, settingsManage
   if (serverSettings.strictTransportSecurity === "always") {
     app.use(hsts({
       maxAge: 10886400000,     // Must be at least 18 weeks to be approved by Google
-      includeSubdomains: true, // Must be enabled to be approved by Google
+      includeSubDomains: true, // Must be enabled to be approved by Google
```

Comment:
API changed



  if ("includeSubdomains" in options) {
    console.warn(
      'Strict-Transport-Security middleware should use `includeSubDomains` instead of `includeSubdomains`. (The correct one has an uppercase "D".)'
    );
  }


https://github.com/helmetjs/helmet/blob/a2549bd4688d9e711aea64a7fdffb07f4136f6cf/middlewares/strict-transport-security/index.ts